### ISSUE TITLE
Suggestion: What about "Axes are NOT weapons?"

### ISSUE THREAD

**Issue by OlaPessoa** [Fourmisain/AxesAreWeapons#16]
could you add a option or make another mod that tweaks the damage of the axe to be the same as the pickaxe + slowing it down even more (same to pickaxe, hoes, shovels) and lastly removing them from the combat section on the creative menu
like the previous versions of the game which in my opinion MAKES A WHOLE LOT MORE OF SENSE

**Comment by Fourmisain** [comment]
The whole idea of the mod was to enable a playstyle that wasn't really possible before - or at least to make it less bumpy.
If you don't like using axes for combat you can always just... well, choose to not use them, you know? 😄
This is all up to you.
However, if you are hosting a multiplayer server and you find players tend to use only one option, consider buffing the alternatives instead.

For example, one of - if not the most - "OP" item in the game is arguably the Elytra.
Once you get one (with Mending and rockets of course), there's basically no reason _not_ to use it, because other travel options are slow, inconvenient and/or expensive to build.
But if the server was modded to have super speed minecarts, people might actually invest and start building travel networks - because once the network is built, it requires no further cost like gunpowder or XP - it just works™.
And if horses are modded to not get stuck in trees so often and be able to somewhat swim, maybe people start using them for exploration as well.

Nerfing options to be worse than vanilla will mostly just annoy and inconvenience your players.
Sometimes fast travel is wanted because it's not about the travel but about saving time to get from A to B.
Sometimes axes are weapons, because the player died and dropped their sword, so they quickly grab the next best thing from their chests.

So long story short: I've got no interest in making the "opposite mod".

If you want to pursue your idea further, I'd honestly suggest to get into modding yourself.
There's [excellent resources to get started](https://fabricmc.net/develop/), like the [Fabric Wiki](https://fabricmc.net/wiki/tutorial:start), along with several Discord servers.
It helps to know some basic Java programming beforehand - but this is also nothing that you can't teach yourself.

In future Minecraft versions, changes like the ones you suggested might also be possible via data packs, 1.21 will e.g. allow to create custom enchantments and modify existing ones entirely through data packs, no modding (in the classical sense) required.

**Comment by OlaPessoa** [comment]
well, what about an option built-in with the mod "Axes are Weapons" then? would that be easy?
+ next time you may be honest and just straight up say something like "sorry but i'm not interested on doing a mod like that" or whatev kind of explanation (if anything i'm not the mod developer here, you are, if anything i need you, not the other way around), no matter what tho, thank you for the response!

**Comment by Fourmisain** [comment]
> by the way next time you may be honest and just straight up say something like "sorry but i'm not interested on doing a mod like that"

Sorry about the wall of text! 😅
I just like laying out my thoughts because sometimes there's discussions to be had and it wouldn't be the first time I change my mind (though I doubt it this time, so that's on me).

> well, what about an option built-in with the mod "Axes are Weapons" then? would that be easy?

Same thing, same answer.
(That straight and honest enough?)

> if anything i'm not the mod developer here, you are, if anything i need you, not the other way around

[Don't let your dreams be dreams, go become a mod dev!](https://i.giphy.com/GcSqyYa2aF8dy.webp)
